### PR TITLE
WIP: Added ability to export chain keys

### PR DESCRIPTION
**Pull request by mitchellolsthoorn**
Needed for #3239, where it is used to export the keys to an android client

Dependencies used:
python-qrcode

### CODE REVIEW

File: TriblerGUI/widgets/settingspage.py [hunk 1]
```diff
@@ -1,4 +1,7 @@
-from PyQt5.QtWidgets import QWidget, QFileDialog
+from PyQt5 import QtGui
+from PyQt5.QtWidgets import QWidget, QLabel, QFileDialog
+from PIL.ImageQt import ImageQt
+import qrcode
```
Comment (devos50) [inline TriblerGUI/widgets/settingspage.py]: Just an idea, we can make this `qrcode` import conditional, something like:

```
try:
    import qrcode
    has_qr = True
except ImportError:
    has_qr = False
```

And show an error if has_qr is `False`?

Comment (mitchellolsthoorn) [reply TriblerGUI/widgets/settingspage.py]: Yeah, that is probably better, so it causes less dependency errors. Do you have an example how I can call an error widget?

Comment (devos50) [reply TriblerGUI/widgets/settingspage.py]: Yes, see https://github.com/Tribler/tribler/blob/devel/TriblerGUI/dialogs/confirmationdialog.py#L46

**Comment by mitchellolsthoorn** [comment]
@devos50, I updated the code to test if the module is present and also included an error for when the keys are missing or are still being created

**Comment by mitchellolsthoorn** [comment]
This PR has become obsolete and is replaced by PR #3358

**Pull request closed without merging**